src/shared/sam/debugger/awsSamDebugConfiguration.ts

```diff
@@ -85,7 +86,27 @@ export function createTemplateAwsSamDebugConfig(
     return response
 }
 
-export function createCodeAwsSamDebugConfig(lambdaHandler: string, projectRoot: string): AwsSamDebuggerConfiguration {
+export function createCodeAwsSamDebugConfig(
+    lambdaHandler: string,
+    projectRoot: string,
+    runtimeFamily?: RuntimeFamily
+): AwsSamDebuggerConfiguration {
+    let runtime: string
+
+    switch (runtimeFamily) {
+        case RuntimeFamily.DotNetCore:
+            runtime = dotNetRuntimes.first()
```

Comment:
can use `nodeJsRuntimes.sort(compareSamLambdaRuntime).first()` , similar to `getDefaultRuntime()`

Reply:
actually you could just use `getDefaultRuntime()` here and don't need the `switch`.

- add a `getRuntimeFamily(string) => RuntimeFamily` function to `debugConfiguration.ts` if needed. 
- then I suppose `getDefaultRuntime()` should be changed to take `RuntimeFamily` enum value.

then this code will be just:

    runtime = getDefaultRuntime(runtimeFamily)

Reply:
actually i just did that here: https://github.com/aws/aws-toolkit-vscode/pull/1060

Reply:
Could've sworn we had that functionality somewhere. Will add, thanks.

Reply:
if you approve https://github.com/aws/aws-toolkit-vscode/pull/1060 then you can just rebase :)